centreon/src/Core/Host/Application/UseCase/AddHost/AddHost.php

```diff
@@ -118,7 +120,9 @@ public function __invoke(AddHostRequest $request, AddHostPresenterInterface $pre
                 $this->linkHostGroups($request, $hostId);
                 $this->linkParentTemplates($request, $hostId);
                 $this->addMacros($request, $hostId);
-                // Note: host is not linked to any ACLsResource
+                if (isset($accessGroups) && $accessGroups !== []) {
```

Comment:
$accessGroups should be initialized with an empty array above line 110, it's instead of doing an isset()

Reply:
Corrected